src/App.tsx

```diff
@@ -1,20 +1,56 @@
+import { FC } from 'react';
+import {
+  Navigate, Route, Routes, useMatch,
+} from 'react-router-dom';
 import { PeoplePage } from './components/PeoplePage';
-import { Navbar } from './components/Navbar';
 
 import './App.scss';
+import { Navbar } from './components/Navbar';
+
+export const App: FC = () => {
+  const match = useMatch('/people/:slug');
 
-export const App = () => {
   return (
-    <div data-cy="app">
-      <Navbar />
+    <>
```

Comment:
looks like a redundant fragment

Reply:
i still can see this redundant fragment :)

Reply:
now, I think, no redundant fragment :]